index.html

```diff
@@ -2110,7 +2110,14 @@ <h2>Capability Delegation</h2>
 {
   "@context": "https://www.w3.org/ns/did/v1.1",
   "id": "did:example:123456789abcdefghi",
-  <span class="comment">...</span>
+  "verificationMethod": [
+    {
+      "id": "did:example:123456789abcdefghi#keys-1",
+      "type": "Ed25519VerificationKey2020",
```

Comment:
Let's move over to using `Multikey` for all the examples since it's now defined.

```suggestion
      "type": "Multikey",
```